portafly/portafly_api.yml

```diff
@@ -74,6 +149,83 @@ components:
         org_name: Umbrella Corp.
         admin_name: Oswell E. Spencer
         apps_count: 3
+    User:
+      title: Root Type for UserData
+      description: A simplified format for a user data.
+      type: object
+      properties:
+        id:
+          description: The user ID number
+          type: integer
+        name:
+          description: The name of the user
+          type: string
+        email:
+          description: The email of the user
+          type: string
+        role:
+          description: The role of the user
+          type: string
+        is_current_user:
+          description: Is current user logged in?
+          type: boolean
+        can_see_users:
+          description: Can the user see other users?
+          type: boolean
+        is_authenticated_through_sso:
+          description: Is the user authenticated through SSO?
+          type: boolean
+        can_delete_user:
+          description: Can current user delete other users?
+          type: boolean
+        can_invite_user:
+          description: Can the current user invite users?
+          type: boolean
+        permissions:
```

Comment:
We can keep this for now I think... I'm still not entirely sure from where they'll come (Keycloak, RH SSO or Porta)